packages/chat-headless/src/slices/conversation.ts

```diff
@@ -2,11 +2,27 @@ import { createSlice, PayloadAction } from "@reduxjs/toolkit";
 import { ConversationState } from "../models/slices/conversation";
 import { Message, MessageNotes } from "@yext/chat-core";
 
+export const STATE_SESSION_STORAGE_KEY = "yext_chat_conversation_state";
+
 export const initialState: ConversationState = {
   messages: [],
   isLoading: false,
 };
 
+/**
+ * Loads the {@link ConversationState} from session storage.
+ */
+export const loadSessionState = (): ConversationState => {
+  if (sessionStorage?.getItem(STATE_SESSION_STORAGE_KEY)) {
+    return JSON.parse(sessionStorage?.getItem(STATE_SESSION_STORAGE_KEY) ?? "");
```

Comment:
Not sure exactly what to do here, it wouldn't compile unless I added a fallback string, since JSON.parse requires a string and getItem can return undefined.

Reply:
could you have it as:
```
if (!sessionStorage) {
console.warn(
      "Session storage is not available. State will not be persisted across page refreshes."
    );
    return initialState;
}
const savedState = sessionStorage.getItem(STATE_SESSION_STORAGE_KEY)
return savedState ? JSON.parse(savedState) : initialState
```

Reply:
Yep, that works nicely. Thanks!